README.md

```diff
@@ -6,49 +6,51 @@ This repository contains algorithms and datastructures implemented in C# for edu
 ---
 
 ## Overview  
-* [Algorithms](./Algorithms/)
```

Comment:
Please revert all changes to this file as it's ok, just Codacy wasn't configured for Markdown.